hooks/persistence-defectdojo/README.md

```diff
@@ -136,3 +136,23 @@ helm upgrade --install dd secureCodeBox/persistence-defectdojo \
 | image.pullPolicy | string | `"IfNotPresent"` | Image pull policy. One of Always, Never, IfNotPresent. Defaults to Always if :latest tag is specified, or IfNotPresent otherwise. More info: https://kubernetes.io/docs/concepts/containers/images#updating-images |
 | image.repository | string | `"docker.io/securecodebox/persistence-defectdojo"` | Hook image repository |
 | image.tag | string | `nil` | Container image tag |
+
+## Running Locally from Source
+For Development, it can be useful to run the Hook locally. You can do so by following these steps:
+
+1. Make sure you have access to a running [DefectDojo](https://github.com/DefectDojo/django-DefectDojo) Instance
+2. [Run a Scan](https://docs.securecodebox.io/docs/getting-started/first-scans) of your choice.
+3. Supply Download Links for the Scan Results (Raw Result and Findings.json). You can e.g., access them from the
+included [Minio Instance](https://docs.securecodebox.io/docs/getting-started/installation/#accessing-the-included-minio-instance)
+and upload them to a GitHub gist.
+
+4. Set the following environment variables
+
+- DEFECTDOJO_URL (e.g http://192.168.0.228:8080);
+- DEFECTDOJO_USERNAME (e.g admin)
+- DEFECTDOJO_APIKEY= (e.g. b09c.., can be fetched from the DefectDojo Settings)
+- IS_DEV=true
+- SCAN_NAME (e.g nmap-scanme.nmap.org)
```

Comment:
What is the `SCAN_NAME` here for? Do i need to add the corresponding Name of the scan which user starts in step 2 or could it be any random scan name? Maybe a comment would help to clearify that :)

Reply:
Done